doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -89,7 +89,7 @@ is `Set` or `Type`. :gdef:`"Hypotheses" <hypothesis>` refers to items that are
 for which the type of their type is `Prop` or `SProp`,
 but these terms are also used interchangeably.
 
-.. coqtop:: out
+.. coqtop:: all
```

Comment:
Is there a good reason to change this from `out` to `all`? It seemed to me that the implementation details were not made to be shown here...

Reply:
It seemed mysterious to hide the small trick needed to print the type.  I could do a parenthetical comment instead.  OTOH, how often do users need to print the type?

Reply:
> OTOH, how often do users need to print the type?

Not often. Only users writing advanced automation would do that. So the example was just meant to highlight the type of the hypotheses not how we compute that.